Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/ColliderTranslationManipulators.h

```diff
@@ -9,17 +9,20 @@
 #pragma once
 
 #include <AzToolsFramework/Manipulators/TranslationManipulators.h>
+#include <Editor/Plugins/Ragdoll/PhysicsSetupManipulatorCommandCallback.h>
 #include <Editor/Plugins/Ragdoll/PhysicsSetupManipulators.h>
 #include <MCore/Source/Command.h>
 #include <MCore/Source/MCoreCommandManager.h>
 
 namespace EMotionFX
 {
-    class PhysicsSetupColliderTranslationManipulators : public PhysicsSetupManipulatorsBase
+    class ColliderTranslationManipulators
```

Comment:
nit: API comment for these public types would be useful

Reply:
Done.